vignettes/articles/04-read-the-outputs.Rmd

```diff
@@ -40,4 +40,218 @@ analysis.
 
 ## Details
 
-**OPEN**
+In this section, we look at the output files and what each of the variables they
+contain mean.
+
+### Shared Outputs: Model Parameters
+
+All result files discussed below contain information on the settings of the
+stress testing model. This means they all contain the following common variables:
+
+* `asset_type_arg`: Asset type for which the analysis was run.
```

Comment:
duplication, can this maybe link to input args instead?

Reply:
done